test/data/matgas/case-6-no-power-limits.m

```diff
@@ -55,5 +55,5 @@
 %% receipt data
 % id junction_id injection_min injection_max injection_nominal is_dispatchable status offer_price name company_name daily_scheduled_flow design_capacity operating_capacity is_firm edi_id
 mgc.receipt = [
-1    1    0    1000    500    1     1  -1000
```

Comment:
why this change?  To match @azlotnik model?

Reply:
Yes.

Reply:
Ok, I had matched this to a model/results he was using in his matlab code.  If that has changed, then I am fine with this modification.